algebra/tests/qdiff/tests.c

```diff
@@ -116,6 +116,18 @@ TEST(group_qvdiff_qpDiffQ, qvdiff_qpDiffQ_complicatedQuaternion)
 }
 
 
+TEST(group_qvdiff_qpDiffQ, qvdiff_qpDiffQ_resultMatTransposed)
```

Comment:
simplification suggestion
```suggestion
TEST(group_qvdiff_qpDiffQ, qvdiff_qpDiffQ_resTrp)
```